cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
         # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
         #targetSizeRatio: .5
+      #compressionMode: none | passive | aggressive | force
+      # For reference: https://docs.ceph.com/docs/nautilus/rados/configuration/bluestore-config-ref/#inline-compression
```

Comment:
Comments in the yaml typically go above the setting.

Reply:
Fixed with the latest update.